### ISSUE TITLE
Warning message during the outlier frame extraction

### ISSUE THREAD

**Issue by Mijar007** [DeepLabCut/DeepLabCut#2659]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I am using DLC in an Anaconda env.
The warning shown below appeared in the Anaconda prompt during the outlier frame extraction. 
It didn't cause an issue for me but I think I should inform you about it, since the message prophesies some problems on a Linux system.

### Operating System

Win 11

### DeepLabCut version

dlc version 3.0.0rc2

### DeepLabCut mode

single animal

### Device type

gpu

### Relevant log output

```shell
C:\ProgramData\anaconda3\envs\DEEPLABCUT\lib\site-packages\threadpoolctl.py:1214: RuntimeWarning:
Found Intel OpenMP ('libiomp') and LLVM OpenMP ('libomp') loaded at
the same time. Both libraries are known to be incompatible and this
can cause random crashes or deadlocks on Linux when loaded in the
same Python program.
Using threadpoolctl may cause crashes or deadlocks. For more
information and possible workarounds, please see
    https://github.com/joblib/threadpoolctl/blob/master/multiple_openmp.md

  warnings.warn(msg, RuntimeWarning)
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by n-poulsen** [comment]
Would you have `intel-openmp` installed in your conda environment? That might be what's causing the issue. Could you please let me know what the output of `pip list` produces in your anaconda prompt (in the DeepLabCut environment)?

**Comment by Mijar007** [comment]
Sure, here is the list, and yes, intel-openmp is installed.
```
Package                       Version
----------------------------- --------------
alabaster                     0.7.16
albumentations                1.4.3
anyio                         4.4.0
app-model                     0.2.7
appdirs                       1.4.4
argon2-cffi                   23.1.0
argon2-cffi-bindings          21.2.0
arrow                         1.3.0
asttokens                     2.4.1
attrs                         23.2.0
Babel                         2.15.0
beautifulsoup4                4.12.3
bleach                        6.1.0
Brotli                        1.1.0
build                         1.2.1
cached-property               1.5.2
cachey                        0.2.1
certifi                       2024.7.4
cffi                          1.16.0
charset-normalizer            3.3.2
click                         8.1.7
cloudpickle                   3.0.0
colorama                      0.4.6
comm                          0.2.2
contourpy                     1.2.1
cycler                        0.12.1
dask                          2024.7.0
dask-expr                     1.1.7
dask-image                    2024.5.3
debugpy                       1.8.2
decorator                     5.1.1
deeplabcut                    3.0.0rc2
defusedxml                    0.7.1
dlclibrary                    0.0.6
docker-pycreds                0.4.0
docstring_parser              0.16
docutils                      0.17.1
einops                        0.8.0
entrypoints                   0.4
exceptiongroup                1.2.0
executing                     2.0.1
fastjsonschema                2.20.0
filelock                      3.15.4
filterpy                      1.4.5
flexcache                     0.3
flexparser                    0.3.1
fonttools                     4.53.1
fqdn                          1.5.1
freetype-py                   2.4.0
fsspec                        2024.6.1
gitdb                         4.0.11
GitPython                     3.1.43
h2                            4.1.0
HeapDict                      1.0.1
hpack                         4.0.0
hsluv                         5.0.4
huggingface-hub               0.23.4
hyperframe                    6.0.1
idna                          3.7
imageio                       2.34.2
imageio-ffmpeg                0.5.1
imagesize                     1.4.1
imgaug                        0.4.0
importlib_metadata            8.0.0
importlib_resources           6.4.0
in-n-out                      0.2.1
intel-openmp                  2021.4.0
ipykernel                     6.29.5
ipython                       8.26.0
ipython_genutils              0.2.0
ipywidgets                    8.1.3
isoduration                   20.11.0
jedi                          0.19.1
Jinja2                        3.1.4
joblib                        1.4.2
jsonpointer                   3.0.0
jsonschema                    4.22.0
jsonschema-specifications     2023.12.1
jupyter                       1.0.0
jupyter_client                7.4.9
jupyter-console               6.6.3
jupyter_core                  5.7.2
jupyter-events                0.10.0
jupyter_server                2.14.1
jupyter_server_terminals      0.5.3
jupyterlab_pygments           0.3.0
jupyterlab_widgets            3.0.11
kiwisolver                    1.4.5
lazy_loader                   0.4
llvmlite                      0.43.0
locket                        1.0.0
magicgui                      0.8.3
markdown-it-py                3.0.0
MarkupSafe                    2.1.5
matplotlib                    3.8.4
matplotlib-inline             0.1.7
mdurl                         0.1.2
mistune                       3.0.2
mkl                           2021.4.0
mpmath                        1.3.0
napari                        0.4.18
napari-console                0.0.9
napari-deeplabcut             0.2.1.7
napari-plugin-engine          0.2.0
napari-svg                    0.2.0
natsort                       8.4.0
nb_conda                      2.2.1
nb_conda_kernels              2.5.1
nbclassic                     1.1.0
nbclient                      0.10.0
nbconvert                     7.16.4
nbformat                      5.10.4
nest_asyncio                  1.6.0
networkx                      3.3
notebook                      6.5.7
notebook_shim                 0.2.4
npe2                          0.7.6
numba                         0.60.0
numexpr                       2.10.0
numpy                         1.26.4
numpydoc                      1.5.0
opencv-python                 4.10.0.84
opencv-python-headless        4.10.0.84
overrides                     7.7.0
packaging                     24.1
pandas                        2.2.2
pandocfilters                 1.5.0
parso                         0.8.4
partd                         1.4.2
patsy                         0.5.6
pickleshare                   0.7.5
pillow                        10.4.0
PIMS                          0.7
Pint                          0.24.1
pip                           24.0
pkgutil_resolve_name          1.3.10
platformdirs                  4.2.2
pooch                         1.8.2
prometheus_client             0.20.0
prompt_toolkit                3.0.47
protobuf                      5.27.2
psutil                        6.0.0
psygnal                       0.11.1
pure-eval                     0.2.2
py-cpuinfo                    9.0.0
pyarrow                       16.1.0
pycocotools                   2.0.8
pyconify                      0.1.6
pycparser                     2.22
pydantic                      1.10.17
pydantic-compat               0.1.2
Pygments                      2.18.0
PyOpenGL                      3.1.7
pyparsing                     3.1.2
pyproject_hooks               1.1.0
PySide6                       6.4.2
PySide6-Addons                6.4.2
PySide6-Essentials            6.4.2
PySocks                       1.7.1
python-dateutil               2.9.0
python-json-logger            2.0.7
pytz                          2024.1
pywin32                       306
pywinpty                      2.0.13
PyYAML                        6.0.1
pyzmq                         26.0.3
QDarkStyle                    3.1
qtconsole                     5.5.2
QtPy                          2.4.1
referencing                   0.35.1
requests                      2.32.3
rfc3339-validator             0.1.4
rfc3986-validator             0.1.1
rich                          13.7.1
rpds-py                       0.19.0
ruamel.yaml                   0.18.6
ruamel.yaml.clib              0.2.8
safetensors                   0.4.3
scikit-image                  0.24.0
scikit-learn                  1.5.1
scipy                         1.10.1
Send2Trash                    1.8.3
sentry-sdk                    2.8.0
setproctitle                  1.3.3
setuptools                    70.1.1
shapely                       2.0.4
shellingham                   1.5.4
shiboken6                     6.4.2
six                           1.16.0
slicerator                    1.1.0
smmap                         5.0.1
sniffio                       1.3.1
snowballstemmer               2.2.0
soupsieve                     2.5
Sphinx                        4.5.0
sphinxcontrib-applehelp       1.0.8
sphinxcontrib-devhelp         1.0.6
sphinxcontrib-htmlhelp        2.0.5
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          1.0.7
sphinxcontrib-serializinghtml 1.1.10
stack-data                    0.6.2
statsmodels                   0.14.2
superqt                       0.6.7
sympy                         1.13.0
tables                        3.8.0
tbb                           2021.13.0
terminado                     0.18.1
threadpoolctl                 3.5.0
tifffile                      2024.7.2
timm                          1.0.7
tinycss2                      1.3.0
tomli                         2.0.1
tomli_w                       1.0.0
toolz                         0.12.1
torch                         2.2.2+cu118
torchvision                   0.17.2+cu118
tornado                       6.4.1
tqdm                          4.66.4
traitlets                     5.14.3
typer                         0.12.3
types-python-dateutil         2.9.0.20240316
typing_extensions             4.12.2
typing-utils                  0.1.0
tzdata                        2024.1
uri-template                  1.3.0
urllib3                       2.2.2
vispy                         0.12.2
wandb                         0.17.4
wcwidth                       0.2.13
webcolors                     24.6.0
webencodings                  0.5.1
websocket-client              1.8.0
wheel                         0.43.0
widgetsnbextension            4.0.11
win-inet-pton                 1.1.0
wrapt                         1.16.0
zipp                          3.19.2
zstandard                     0.22.0
```

**Comment by MMathisLab** [comment]
can you do a fresh install @Mijar007 - as intel- should not be there and is :)

**Comment by Mijar007** [comment]
Sorry for the delay, I was on vacation.
I freshly installed DLC (**now dlc version 3.0.0rc3**)

Since I have to do a three step installation (CUDA), I checked the pip list after every step. 
Here are my steps:
1) `conda env create -f DEEPLABCUT.yaml`
`conda activate DEEPLABCUT`
2) `conda install -c conda-forge pytables==3.8.0`
3) `pip install torch==2.2.2 torchvision==0.17.2 --index-url https://download.pytorch.org/whl/cu118`

'intel-openmp' was not listed after each step.

However, as I extracted the frames for the first training data set, I got again the warning (I also added the prompts before the warning:

```
INFO:console:Extracting frames based on kmeans ...
INFO:console:Kmeans-quantization based extracting of frames from
INFO:console:
INFO:console:0.0
INFO:console:
INFO:console: seconds to
INFO:console:
INFO:console:71.27
INFO:console:
INFO:console: seconds.
INFO:console:Extracting and downsampling...
INFO:console:
INFO:console:1709
0it [00:00, ?it/s]INFO:console:
INFO:console: frames from the video.
1709it [00:22, 74.81it/s]
INFO:console:Kmeans clustering ... (this might take a while)

C:\ProgramData\anaconda3\envs\DEEPLABCUT\lib\site-packages\threadpoolctl.py:1214: RuntimeWarning:
Found Intel OpenMP ('libiomp') and LLVM OpenMP ('libomp') loaded at
the same time. Both libraries are known to be incompatible and this
can cause random crashes or deadlocks on Linux when loaded in the
same Python program.
Using threadpoolctl may cause crashes or deadlocks. For more
information and possible workarounds, please see
    https://github.com/joblib/threadpoolctl/blob/master/multiple_openmp.md

  warnings.warn(msg, RuntimeWarning)
```

I closed DLC and checked again the pip list. 'intel-openmp' is still not listed:
Package                       Version
----------------------------- --------------
alabaster                     0.7.16
albumentations                1.4.3
anyio                         4.4.0
app-model                     0.2.8
appdirs                       1.4.4
argon2-cffi                   23.1.0
argon2-cffi-bindings          21.2.0
arrow                         1.3.0
asttokens                     2.4.1
attrs                         23.2.0
Babel                         2.15.0
beautifulsoup4                4.12.3
bleach                        6.1.0
Brotli                        1.1.0
build                         1.2.1
cached-property               1.5.2
cachey                        0.2.1
certifi                       2024.7.4
cffi                          1.16.0
charset-normalizer            3.3.2
click                         8.1.7
cloudpickle                   3.0.0
colorama                      0.4.6
comm                          0.2.2
contourpy                     1.2.1
cycler                        0.12.1
dask                          2024.7.1
dask-expr                     1.1.9
dask-image                    2024.5.3
debugpy                       1.8.2
decorator                     5.1.1
deeplabcut                    3.0.0rc3
defusedxml                    0.7.1
dlclibrary                    0.0.6
docker-pycreds                0.4.0
docstring_parser              0.16
docutils                      0.17.1
einops                        0.8.0
entrypoints                   0.4
exceptiongroup                1.2.2
executing                     2.0.1
fastjsonschema                2.20.0
filelock                      3.15.4
filterpy                      1.4.5
flexcache                     0.3
flexparser                    0.3.1
fonttools                     4.53.1
fqdn                          1.5.1
freetype-py                   2.4.0
fsspec                        2024.6.1
gitdb                         4.0.11
GitPython                     3.1.43
h2                            4.1.0
HeapDict                      1.0.1
hpack                         4.0.0
hsluv                         5.0.4
huggingface-hub               0.24.3
hyperframe                    6.0.1
idna                          3.7
imageio                       2.34.2
imageio-ffmpeg                0.5.1
imagesize                     1.4.1
imgaug                        0.4.0
importlib_metadata            8.2.0
importlib_resources           6.4.0
in-n-out                      0.2.1
ipykernel                     6.29.5
ipython                       8.26.0
ipython_genutils              0.2.0
ipywidgets                    8.1.3
isoduration                   20.11.0
jedi                          0.19.1
Jinja2                        3.1.4
joblib                        1.4.2
jsonpointer                   3.0.0
jsonschema                    4.23.0
jsonschema-specifications     2023.12.1
jupyter                       1.0.0
jupyter_client                7.4.9
jupyter-console               6.6.3
jupyter_core                  5.7.2
jupyter-events                0.10.0
jupyter_server                2.14.2
jupyter_server_terminals      0.5.3
jupyterlab_pygments           0.3.0
jupyterlab_widgets            3.0.11
kiwisolver                    1.4.5
lazy_loader                   0.4
llvmlite                      0.43.0
locket                        1.0.0
magicgui                      0.8.3
markdown-it-py                3.0.0
MarkupSafe                    2.1.5
matplotlib                    3.8.4
matplotlib-inline             0.1.7
mdurl                         0.1.2
mistune                       3.0.2
mpmath                        1.3.0
napari                        0.4.18
napari-console                0.0.9
napari-deeplabcut             0.2.1.7
napari-plugin-engine          0.2.0
napari-svg                    0.2.0
natsort                       8.4.0
nb_conda                      2.2.1
nb_conda_kernels              2.5.1
nbclassic                     1.1.0
nbclient                      0.10.0
nbconvert                     7.16.4
nbformat                      5.10.4
nest_asyncio                  1.6.0
networkx                      3.3
notebook                      6.5.7
notebook_shim                 0.2.4
npe2                          0.7.7
numba                         0.60.0
numexpr                       2.10.0
numpy                         1.26.4
numpydoc                      1.5.0
opencv-python                 4.10.0.84
opencv-python-headless        4.10.0.84
overrides                     7.7.0
packaging                     24.1
pandas                        2.2.2
pandocfilters                 1.5.0
parso                         0.8.4
partd                         1.4.2
patsy                         0.5.6
pickleshare                   0.7.5
pillow                        10.4.0
PIMS                          0.7
Pint                          0.24.3
pip                           24.0
pkgutil_resolve_name          1.3.10
platformdirs                  4.2.2
pooch                         1.8.2
prometheus_client             0.20.0
prompt_toolkit                3.0.47
protobuf                      5.27.2
psutil                        6.0.0
psygnal                       0.11.1
pure_eval                     0.2.3
py-cpuinfo                    9.0.0
pyarrow                       17.0.0
pycocotools                   2.0.8
pyconify                      0.1.6
pycparser                     2.22
pydantic                      1.10.17
pydantic-compat               0.1.2
Pygments                      2.18.0
PyOpenGL                      3.1.7
pyparsing                     3.1.2
pyproject_hooks               1.1.0
PySide6                       6.4.2
PySide6-Addons                6.4.2
PySide6-Essentials            6.4.2
PySocks                       1.7.1
python-dateutil               2.9.0
python-json-logger            2.0.7
pytz                          2024.1
pywin32                       306
pywinpty                      2.0.13
PyYAML                        6.0.1
pyzmq                         26.0.3
QDarkStyle                    3.1
qtconsole                     5.5.2
QtPy                          2.4.1
referencing                   0.35.1
requests                      2.32.3
rfc3339-validator             0.1.4
rfc3986-validator             0.1.1
rich                          13.7.1
rpds-py                       0.19.1
ruamel.yaml                   0.18.6
ruamel.yaml.clib              0.2.8
safetensors                   0.4.3
scikit-image                  0.24.0
scikit-learn                  1.5.1
scipy                         1.10.1
Send2Trash                    1.8.3
sentry-sdk                    2.11.0
setproctitle                  1.3.3
setuptools                    71.0.4
shapely                       2.0.5
shellingham                   1.5.4
shiboken6                     6.4.2
six                           1.16.0
slicerator                    1.1.0
smmap                         5.0.1
sniffio                       1.3.1
snowballstemmer               2.2.0
soupsieve                     2.5
Sphinx                        4.5.0
sphinxcontrib-applehelp       2.0.0
sphinxcontrib-devhelp         2.0.0
sphinxcontrib-htmlhelp        2.1.0
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          2.0.0
sphinxcontrib-serializinghtml 2.0.0
stack-data                    0.6.2
statsmodels                   0.14.2
superqt                       0.6.7
sympy                         1.13.1
tables                        3.8.0
terminado                     0.18.1
threadpoolctl                 3.5.0
tifffile                      2024.7.24
timm                          1.0.8
tinycss2                      1.3.0
tomli                         2.0.1
tomli_w                       1.0.0
toolz                         0.12.1
torch                         2.2.2+cu118
torchvision                   0.17.2+cu118
tornado                       6.4.1
tqdm                          4.66.4
traitlets                     5.14.3
typer                         0.12.3
types-python-dateutil         2.9.0.20240316
typing_extensions             4.12.2
typing-utils                  0.1.0
tzdata                        2024.1
uri-template                  1.3.0
urllib3                       2.2.2
vispy                         0.12.2
wandb                         0.17.5
wcwidth                       0.2.13
webcolors                     24.6.0
webencodings                  0.5.1
websocket-client              1.8.0
wheel                         0.43.0
widgetsnbextension            4.0.11
win-inet-pton                 1.1.0
wrapt                         1.16.0
zipp                          3.19.2
zstandard                     0.23.0

**Comment by Mijar007** [comment]
And again, the warning also appeared during the outlier frame extraction. Not surprising for me, I guess the same functions are used.

**Comment by n-poulsen** [comment]
I think the other package (LLVM OpenMP) is this `llvmlite` package - there may not be any issues, but if you run into any you can follow [this thread](https://stackoverflow.com/a/78065862) and remove one of them.

You can also try to re-create a conda environment from scratch, and if the same warning is printed please let me know (as well as the exact commands used to create the environment). I'll try to replicate it then
